diff --git a/node_modules/expo-file-system/ios/FileSystemModule.swift b/node_modules/expo-file-system/ios/FileSystemModule.swift
index d564ae7..6239511 100644
--- a/node_modules/expo-file-system/ios/FileSystemModule.swift
+++ b/node_modules/expo-file-system/ios/FileSystemModule.swift
@@ -37,7 +37,7 @@ public final class FileSystemModule: Module {
     Events(EVENT_DOWNLOAD_PROGRESS, EVENT_UPLOAD_PROGRESS)
 
     AsyncFunction("getInfoAsync") { (url: URL, options: InfoOptions, promise: Promise) in
-      let optionsDict = options.toDictionary(appContext: appContext)
+      let optionsDict = options.toDictionary()
       switch url.scheme {
       case "file":
         EXFileSystemLocalFileHandler.getInfoForFile(url, withOptions: optionsDict, resolver: promise.resolver, rejecter: promise.legacyRejecter)
diff --git a/node_modules/expo-file-system/ios/Next/FileSystemNextModule.swift b/node_modules/expo-file-system/ios/Next/FileSystemNextModule.swift
index 2fef7d7..727daa6 100644
--- a/node_modules/expo-file-system/ios/Next/FileSystemNextModule.swift
+++ b/node_modules/expo-file-system/ios/Next/FileSystemNextModule.swift
@@ -197,11 +197,7 @@ public final class FileSystemNextModule: Module {
       return [:]
     }
     var result: [String: String] = [:]
-    for appGroup in appContext.appCodeSignEntitlements.appGroups ?? [] {
-      if let directory = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: appGroup) {
-        result[appGroup] = directory.standardizedFileURL.path
-      }
-    }
+   
     return result
   }
 }
