diff --git a/node_modules/react-native-face-pile/src/index.js b/node_modules/react-native-face-pile/src/index.js
index 76562d7..3fc788d 100644
--- a/node_modules/react-native-face-pile/src/index.js
+++ b/node_modules/react-native-face-pile/src/index.js
@@ -1,6 +1,9 @@
 import React, { PureComponent } from 'react'
 import PropTypes from 'prop-types'
-import { View, Text, Image, StyleSheet, Animated } from 'react-native'
+import { View, Text, StyleSheet, Animated } from 'react-native'
+import FastImage from "react-native-fast-image";
+import { createImageProgress } from "react-native-image-progress";
+const Image = createImageProgress(FastImage);
 
 const styles = StyleSheet.create({
   container: {
@@ -49,7 +52,10 @@ class Circle extends PureComponent {
             },
             imageStyle
           ]}
-          source={{ uri: face.imageUrl }}
+          source={{ uri: face.imageUrl,
+            priority: FastImage.priority.high,
+            cache: FastImage.cacheControl.immutable,
+           }}
         />
       </Animated.View>
     )
@@ -103,6 +109,53 @@ export default class FacePile extends PureComponent {
     hideOverflow: false,
   }
 
+  _renderOverflowCircleZero = overflow => {
+    const {
+      circleStyle,
+      overflowStyle,
+      overflowLabelStyle,
+      circleSize,
+      offset
+    } = this.props
+    
+    const innerCircleSize = circleSize * 1.8
+    const marginLeft = (circleSize * offset) - circleSize / 1.6
+
+    return (
+      <View
+        style={[
+          styles.circle,
+          circleStyle
+        ]}
+      >
+        <View
+          style={[
+            styles.overflow,
+            {
+              width: innerCircleSize,
+              height: innerCircleSize,
+              borderRadius: circleSize,
+              marginLeft: marginLeft
+            },
+            overflowStyle
+          ]}
+        >
+          <Text
+            style={[
+              styles.overflowLabel,
+              {
+                fontSize: circleSize * 0.7
+              },
+              overflowLabelStyle
+            ]}
+          >
+            {overflow}
+          </Text>
+        </View>
+      </View>
+    )
+  }
+
   _renderOverflowCircle = overflow => {
     const {
       circleStyle,
@@ -174,7 +227,8 @@ export default class FacePile extends PureComponent {
 
     return (
       <View style={[styles.container, containerStyle]}>
-        {overflow > 0 && !hideOverflow && this._renderOverflowCircle(overflow)}
+        {overflow > 0 && this._renderOverflowCircle(overflow)}
+        {facesToRender <= 0 && this._renderOverflowCircleZero(overflow)}
         {Array.isArray(facesToRender) && facesToRender.map(this._renderFace)}
       </View>
     )
