diff --git a/node_modules/react-native-permissions-modal/lib/components/PermissionItem/PermissionItem.js b/node_modules/react-native-permissions-modal/lib/components/PermissionItem/PermissionItem.js
index 948d68a..b6c483f 100644
--- a/node_modules/react-native-permissions-modal/lib/components/PermissionItem/PermissionItem.js
+++ b/node_modules/react-native-permissions-modal/lib/components/PermissionItem/PermissionItem.js
@@ -14,15 +14,15 @@ const chevronIcon = require("../../local-assets/chevron.png");
 
 const PermissionItem = props => {
   const {
-    title,
-    source,
-    subtitle,
+    title = "Location",
+    source = require("../../local-assets/placeholder.png"),
+    subtitle = "Give us the this permission to access full features",
     iconComponent,
     titleTextStyle,
-    ImageComponent,
-    iconStatusColor,
+    ImageComponent = Image,
+    iconStatusColor = "green",
     subtitleTextStyle,
-    iconContainerBackgroundColor
+    iconContainerBackgroundColor = "#f7f7f7"
   } = props;
   return (
     <Ripple
@@ -59,13 +59,4 @@ PermissionItem.propTypes = {
   iconContainerBackgroundColor: PropTypes.string
 };
 
-PermissionItem.defaultProps = {
-  title: "Location",
-  ImageComponent: Image,
-  iconStatusColor: "green",
-  iconContainerBackgroundColor: "#f7f7f7",
-  source: require("../../local-assets/placeholder.png"),
-  subtitle: "Give us the this permission to access full features"
-};
-
 export default PermissionItem;
diff --git a/node_modules/react-native-permissions-modal/lib/components/PermissionItem/PermissionItem.style.js b/node_modules/react-native-permissions-modal/lib/components/PermissionItem/PermissionItem.style.js
index 3f3f5ba..7406e44 100644
--- a/node_modules/react-native-permissions-modal/lib/components/PermissionItem/PermissionItem.style.js
+++ b/node_modules/react-native-permissions-modal/lib/components/PermissionItem/PermissionItem.style.js
@@ -37,12 +37,12 @@ export default {
     width: width * 0.7
   },
   titleTextStyle: {
-    fontSize: 14,
+    fontSize: 15,
     color: "#6a6a6a",
     fontWeight: "600"
   },
   subtitleTextStyle: {
-    fontSize: 12,
+    fontSize: 13,
     color: "#949494"
   },
   contextContainer: {
diff --git a/node_modules/react-native-permissions-modal/lib/components/PermissionModal/PermissionModal.js b/node_modules/react-native-permissions-modal/lib/components/PermissionModal/PermissionModal.js
index a9a8523..ea9b56a 100644
--- a/node_modules/react-native-permissions-modal/lib/components/PermissionModal/PermissionModal.js
+++ b/node_modules/react-native-permissions-modal/lib/components/PermissionModal/PermissionModal.js
@@ -23,12 +23,16 @@ export default class PermissionModal extends React.PureComponent {
   };
 
   renderContent = () => {
-    const { title, subtitle } = this.props;
+    const { 
+      title = "", 
+      subtitle = "" 
+    } = this.props;
+    
     return (
       <View style={styles.content}>
         <View>
           <Text style={styles.titleStyle}>{title}</Text>
-          <Text numberOfLines={3} style={styles.subtitleStyle}>
+          <Text numberOfLines={5} style={styles.subtitleStyle}>
             {subtitle}
           </Text>
         </View>
@@ -56,8 +60,3 @@ PermissionModal.propTypes = {
   subtitle: PropTypes.string
 };
 
-PermissionModal.defaultProps = {
-  title: "Permissions",
-  subtitle:
-    "Phasellus id scelerisque sem. Vivamus consequat leo turpis, vitae sagittis lacus efficitur eu.Phasellus id scelerisque sem. Vivamus consequat leo turpis, vitae sagittis lacus efficitur eu."
-};
diff --git a/node_modules/react-native-permissions-modal/lib/components/PermissionModal/PermissionModal.style.js b/node_modules/react-native-permissions-modal/lib/components/PermissionModal/PermissionModal.style.js
index 84250d3..7d2363b 100644
--- a/node_modules/react-native-permissions-modal/lib/components/PermissionModal/PermissionModal.style.js
+++ b/node_modules/react-native-permissions-modal/lib/components/PermissionModal/PermissionModal.style.js
@@ -15,11 +15,10 @@ export default StyleSheet.create({
   },
   subtitleStyle: {
     fontSize: 15,
-    fontSize: 14,
-    color: "#666",
+    color: "#333",
     lineHeight: 22,
     paddingTop: 10,
     paddingBottom: 10,
-    fontWeight: "200"
+    fontWeight: "400"
   }
 });
