diff --git a/node_modules/react-native-permissions-modal/lib/components/PermissionItem/PermissionItem.js b/node_modules/react-native-permissions-modal/lib/components/PermissionItem/PermissionItem.js
index 948d68a..ac4cbbb 100644
--- a/node_modules/react-native-permissions-modal/lib/components/PermissionItem/PermissionItem.js
+++ b/node_modules/react-native-permissions-modal/lib/components/PermissionItem/PermissionItem.js
@@ -1,6 +1,6 @@
 import React from "react";
 import PropTypes from "prop-types";
-import { Text, View, Image } from "react-native";
+import { Text, View } from "react-native";
 import Ripple from "react-native-material-ripple";
 /**
  * ? Local Imports
@@ -59,13 +59,5 @@ PermissionItem.propTypes = {
   iconContainerBackgroundColor: PropTypes.string
 };
 
-PermissionItem.defaultProps = {
-  title: "Location",
-  ImageComponent: Image,
-  iconStatusColor: "green",
-  iconContainerBackgroundColor: "#f7f7f7",
-  source: require("../../local-assets/placeholder.png"),
-  subtitle: "Give us the this permission to access full features"
-};
 
 export default PermissionItem;
diff --git a/node_modules/react-native-permissions-modal/lib/components/PermissionModal/PermissionModal.js b/node_modules/react-native-permissions-modal/lib/components/PermissionModal/PermissionModal.js
index a9a8523..1a66cc4 100644
--- a/node_modules/react-native-permissions-modal/lib/components/PermissionModal/PermissionModal.js
+++ b/node_modules/react-native-permissions-modal/lib/components/PermissionModal/PermissionModal.js
@@ -55,9 +55,3 @@ PermissionModal.propTypes = {
   title: PropTypes.string,
   subtitle: PropTypes.string
 };
-
-PermissionModal.defaultProps = {
-  title: "Permissions",
-  subtitle:
-    "Phasellus id scelerisque sem. Vivamus consequat leo turpis, vitae sagittis lacus efficitur eu.Phasellus id scelerisque sem. Vivamus consequat leo turpis, vitae sagittis lacus efficitur eu."
-};
